warehouse/tunnelling/model.go

```diff
@@ -0,0 +1,72 @@
+package tunnelling
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	stunnel "github.com/rudderlabs/sql-tunnels/driver/ssh"
+)
+
+var ErrMissingKey = errors.New("missing mandatory key")
+var ErrUnexpectedType = errors.New("unexpected type")
+
+const (
+	SSHForward Type = "ssh_forward"
```

Comment:
Is this for future reference since we are not using it now?

Reply:
I'll remove this. Remanent of past